src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,28 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents a command that Duke will operate.
+ */
```

Comment:
I think I understand why you would want to have isExit()  as part of your interface, perhaps one day there may be another command that allows the user to exit besides with the ExitCommand, though unlikely. I would remove it and have just the ExitCommand have an `boolean isExit()` method. Then all your `*Command` classes will have less boilerplate code (: